app/renderer/components/main/braveryPanel.js

```diff
@@ -184,164 +240,692 @@ class BraveryPanel extends ImmutableComponent {
+              </div>
+            </section>
             : null
           }
-          <hr className='braveryBottomSplitter' />
-          <div className='braveryPanelFooter'>
-            <span className='clickable' onClick={this.onEditGlobal} data-l10n-id='editBraveryGlobalSettings' />
-            <span className='reloadButton clickable' onClick={this.onReload}>
-              <div className='reloadText' data-l10n-id='reload' />
-              <div className='fa fa-repeat' />
-            </span>
+          <hr className={css(
+            styles.braveryPanel__body__hr,
+            styles.braveryPanel__body__hr_splitter,
+            compactBraveryPanel && styles.braveryPanel_compact__body__hr
+          )} />
+          <div className={css(styles.braveryPanel__body__footer)}>
+            <span className={css(
+              styles.braveryPanel__body__footer__edit,
+              styles.braveryPanel__body__footer__edit_clickable,
+              compactBraveryPanel && styles.braveryPanel_compact__body__footer__edit
+            )}
+              onClick={this.onEditGlobal}
+              data-l10n-id='editBraveryGlobalSettings'
+            />
+            <div className={css(
+              styles.braveryPanel__body__footer__reload,
+              styles.braveryPanel__body__footer__reload_clickable
+            )}
+              onClick={this.onReload}>
+              <span className={css(styles.braveryPanel__body__footer__reload__text)} data-l10n-id='reload' />
+              <span className='fa fa-repeat' />
+            </div>
           </div>
-        </div>
+        </section>
       </div>
     </Dialog>
   }
 }
 
+const displayHost = {
+  fontSize: '20px',
+  fontWeight: 'normal',
+  textOverflow: 'ellipsis',
+  whiteSpace: 'nowrap',
+  overflow: 'hidden'
+}
+const editGlobalMarginBottom = '.25rem'
+
+const gridStyles = StyleSheet.create({
+  row1col1: {
+    gridRow: 1,
+    gridColumn: 1
+  },
+  row1col2: {
+    gridRow: 1,
+    gridColumn: 2
+  },
+  row1col3: {
+    gridRow: 1,
+    gridColumn: 3
+  },
+  row1col4: {
+    gridRow: 1,
+    gridColumn: 4
+  },
+  row2col1: {
+    gridRow: 2,
+    gridColumn: 1
+  },
+  row2col2: {
+    gridRow: 2,
+    gridColumn: 2
+  },
+  row2col3: {
+    gridRow: 2,
+    gridColumn: 3
+  },
+  row2col4: {
+    gridRow: 2,
+    gridColumn: 4
+  },
+  row3col1: {
+    gridRow: 3,
+    gridColumn: 1
+  },
+  row3col2: {
+    gridRow: 3,
+    gridColumn: 2
+  },
+  row4col1: {
+    gridRow: 4,
+    gridColumn: 1
+  },
+  row4col2: {
+    gridRow: 4,
+    gridColumn: 2
+  },
+  row5col1: {
+    gridRow: 5,
+    gridColumn: 1
+  },
+  row6col1: {
+    gridRow: 6,
+    gridColumn: 1
+  },
+  row7col1: {
+    gridRow: 7,
+    gridColumn: 1
+  },
+  row8col1: {
+    gridRow: 8,
+    gridColumn: 1
+  }
+})
+
+const styles = StyleSheet.create({
+  braveryPanel: {
+    padding: 0,
+    width: '500px',
+    right: '20px',
+    userSelect: 'none',
+    cursor: 'default',
+    color: '#3B3B3B',
```

Comment:
can we move it to `global.js` and so other colors? you can copy browserButton way and create an object like:

```js
braveryPanel: {
  color: '#3B3B3B'
}
```